### PR TITLE
Liz's HW4

### PR DESCRIPTION
**Pull request by lizlee0225**
Hope you had a good weekend!

### CODE REVIEW

File: D04_ex00.py [hunk 1]
```diff
@@ -10,20 +10,37 @@
...
+			count += 1
+			continue
+		if n == x:
+			print ('You got the number!')
+			break
+		elif n > x:
+			print('Your number is too high.')
+			count += 1
+			continue
+		elif n < x:
```
Comment (danielsgriffin) [inline D04_ex00.py]: Can't this be else?

File: D04_ex00.py [hunk 2]
```diff
@@ -10,20 +10,37 @@
...
+			print('Your number is too low.')
+			count += 1
```
Comment (danielsgriffin) [inline D04_ex00.py]: Is there a way to not have to write `count += 1` in so many places?

File: HW04_ch07_ex02.py [hunk 3]
```diff
@@ -20,16 +20,26 @@
 # the last expression it evaluated.
 
 ###############################################################################
-# Imports
+import math
+
+def eval_loop():
```
Comment (danielsgriffin) [inline HW04_ch07_ex02.py]: What if I type done on the first iteration?

File: HW04_ch08_ex05.py [hunk 4]
```diff
@@ -2,3 +2,33 @@
 # See Chapter 8: Strings Exercise 5 for the task.
 # Please do provide function calls that test/demonstrate your
 # function.
+
+
+def rotate_word(s, n):
+	y = 0
```
Comment (danielsgriffin) [inline HW04_ch08_ex05.py]: the meaning/use of this variable is not readily apparent from its name

File: HW04_ch08_ex05.py [hunk 5]
```diff
@@ -2,3 +2,33 @@
 # See Chapter 8: Strings Exercise 5 for the task.
 # Please do provide function calls that test/demonstrate your
 # function.
+
+
+def rotate_word(s, n):
```
Comment (danielsgriffin) [inline HW04_ch08_ex05.py]: Good job.

**Comment by danielsgriffin** [comment]
Good job.

File: D04_ex00.py [hunk 6]
```diff
@@ -10,20 +10,37 @@
...
-# Body
-
-
+import random
+
+def random_game():
+	x = random.randint(0,25)
+	n = ''
+	count = 0
+	while count < 5:
```
Comment (sberthely) [inline D04_ex00.py]: You could probably increment the count in this place before checking it's value.

File: HW04_ch07_ex02.py [hunk 7]
```diff
@@ -20,16 +20,26 @@
...
+	user_input = ''
+	while user_input != 'done':
+		user_input = input('What would you like to calculate? ')
+		if user_input == 'done':
+			break
+		user_answer = eval(user_input)
+		print(user_answer)
```
Comment (sberthely) [inline HW04_ch07_ex02.py]: you can use the print like this: 
`print(eval(user_input))`

File: HW04_ch08_ex00.py [hunk 8]
```diff
@@ -16,18 +16,23 @@
...
 
 ###############################################################################
 def main():
 
     # Remove print("Hello World!") and add several functions calls to count()
     # below, passing various strings and letters
-    print("Hello World!")
+    count('Hello World!', 'o')
+    count('Liz Lee', 's')
+    count('Ahhhhh', 'h')
```
Comment (sberthely) [inline HW04_ch08_ex00.py]: GREAT, I used your code to correct mine! =)

**Pull request closed without merging**